sample-chat/src/main/java/com/quickblox/sample/chat/ui/activity/SelectUsersActivity.java

```diff
@@ -136,21 +141,41 @@ private void loadUsersFromQb() {
         progressBar.setVisibility(View.VISIBLE);
         QBUsers.getUsersByTags(tags, null).performAsync(new QBEntityCallback<ArrayList<QBUser>>() {
             @Override
-            public void onSuccess(ArrayList<QBUser> result, Bundle params) {
-                QBChatDialog dialog = (QBChatDialog) getIntent().getSerializableExtra(EXTRA_QB_DIALOG);
-
-                usersAdapter = new CheckboxUsersAdapter(SelectUsersActivity.this, result);
-                if (dialog != null) {
-                    usersAdapter.addSelectedUsers(dialog.getOccupants());
-                }
-                usersListView.setAdapter(usersAdapter);
+            public void onSuccess(ArrayList<QBUser> users, Bundle params) {
+                setUserList(users);
+                QBChatDialog oldDialog = (QBChatDialog) getIntent().getSerializableExtra(EXTRA_QB_DIALOG);
+                updateDialog(oldDialog);
+            }
 
+            @Override
+            public void onError(QBResponseException e) {
+                showErrorSnackbar(R.string.select_users_get_users_error, e,
+                        new View.OnClickListener() {
+                            @Override
+                            public void onClick(View v) {
+                                loadUsersFromQb();
+                            }
+                        });
                 progressBar.setVisibility(View.GONE);
             }
+        });
+    }
+
+    private void setUserList (List<QBUser> userList) {
+        this.users = userList;
+    }
+
+    private void updateDialog(QBChatDialog qbChatDialog) {
```

Comment:
incorrect method name, "update" means that you're putting changes to server, so it should be "load"/"get"/"refresh" depending on overall code style